rebar.config

```diff
@@ -11,11 +11,12 @@
 {eunit_opts, [verbose]}.
 
 {deps, [
+        {riak_pb, ".*", {git, "git://github.com/basho/riak_pb", {branch, "feature/jrd/index_body"}}},
+        {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "feature/jrd/index_body"}}},
```

Comment:
Yep, I didn’t review rebar.config changes, will have to tweak these, thanks.

-John

> On Oct 14, 2015, at 4:03 PM, Mark Allen <notifications@github.com> wrote:
> 
> In rebar.config <https://github.com/basho/riak_test/pull/888#discussion_r42045153>:
> 
> > @@ -11,11 +11,12 @@
> >  {eunit_opts, [verbose]}.
> >  
> >  {deps, [
> > +        {riak_pb, ".*", {git, "git://github.com/basho/riak_pb", {branch, "feature/jrd/index_body"}}},
> > +        {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "feature/jrd/index_body"}}},
> We talked about some other branches in HipChat :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/basho/riak_test/pull/888/files#r42045153>.
>